docs/book/v2/links-and-resources.md

```diff
@@ -198,4 +198,42 @@ $resource = $resource->withLink($link);
 $resource = $resource->withoutLink($link);
 ```
 
+To maintain consistency in the structure of the response, you may choose to embed both non-empty and empty collections within the `_embedded` section. This can be achieved by enabling the `embed-empty-collections` configuration option.
```

Comment:
Add a note here that this feature is available since version 2.7.0.

(Otherwise, we'll have folks filing issues wondering why the configuration does not work... only to find out later they're on an earlier version.)

This can be done as follows:

```markdown
> INFO: Available since 2.7.0
> 
> The `embed-empty-collections` configuration option is available starting with version 2.7.0.
```